pkg/collectors/builder.go

```diff
@@ -20,16 +20,16 @@ package collectors
 import (
 	"strings"
 
-	apps "k8s.io/api/apps/v1beta1"
-	autoscaling "k8s.io/api/autoscaling/v2beta1"
-	batchv1 "k8s.io/api/batch/v1"
-	batchv1beta1 "k8s.io/api/batch/v1beta1"
-	extensions "k8s.io/api/extensions/v1beta1"
+	// 	apps "k8s.io/api/apps/v1beta1"
```

Comment:
I have done the refactor only for service.go for now. Thereby one can ignore this change.